packages/lib/src/components/Card/types.ts

```diff
@@ -410,6 +410,13 @@ export interface CardConfiguration extends UIElementProps {
      */
     supportedShopperInteractions?: string[];
 
+    /**
+     * For the PAN field: If the card number is now at one of it's valid lengths and the formatting process has added a separator
+     * to the end of the string... delete the separator
+     * - merchant set config option. Defaults to true
+     */
+    trimTrailingSeparator?: boolean;
+
```

Comment:
Where is this property being used? I see it being added but I don't see where it is being used. 

Also, why are we adding this? was this requested by a merchant? I didn't fully understand its use-case by the description